pkg/garden/app/treaty.hoon

```diff
@@ -314,8 +326,11 @@
   ++  kick
     (kick:io path ~)
   ++  give
+    ^-  (list card)
     =/  t=treaty  (~(got by sovereign) desk)
-    (fact:io (treaty:cg t) /sovereign path ~)
+    :~  (fact:io (treaty-update:cg %add t) /treaties ~)
```

Comment:
Looking at `+on-watch`, the `/sovereign` path here seems like it was an erroneous leftover. We don't distinguish between "subscribing to treaties published by the host" and "all treaties known to the host" anymore, do we? Just making sure I understand this correctly.